tests/director/api/validation_test.go

```diff
@@ -331,6 +331,46 @@ func fixEventAPIDefinitionInput() graphql.EventDefinitionInput {
 		}}
 }
 
+func fixEventAPIDefinitionInputWithName(name string) graphql.EventDefinitionInput {
```

Comment:
Please move all fixtures added to this file to `fixtures.go` if you really need them. Are they really necessary, though? To test validation, all we need is one invalid field. I think it's easier to prepare invalid input inline.